android/app/src/main/java/com/microsoft/reacttestapp/TestApp.kt

```diff
@@ -20,12 +22,18 @@ class TestApp : Application(), HasAndroidInjector, ReactApplication {
     override fun onCreate() {
         super.onCreate()
 
-        val testAppComponent = DaggerTestAppComponent.builder()
-            .binds(this)
-            .build()
+        val eventConsumers = PackageList(this).packages
+            .filter { it is ReactTestAppLifecycleEvents }
+            .map { it as ReactTestAppLifecycleEvents }
+
+        eventConsumers.forEach { it.onTestAppCreated() }
 
+        val testAppComponent = DaggerTestAppComponent.builder()
+                .binds(this)
+                .build()
```

Comment:
Did you mean to indent here?
```suggestion
            .binds(this)
            .build()
```